### PR TITLE
feat: crowdin

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]

**Comment by alexgarel** [comment]
@teolemon at what point are we with this ?

File: po/common/ar.po [hunk 1]
```diff
@@ -3817,7 +3826,8 @@ msgstr "من أجل الحصول على النتائج ذات الصلة، ير
 # "product photos" in this sentence means photos for many products, not just one product
 msgctxt "import_photos_title"
 msgid "Import product photos"
-msgstr "استيراد صور المنتج"
+msgstr "عربي\n"
```
Comment (teolemon) [inline po/common/ar.po:3829]:
```suggestion
msgstr ""
```

File: po/common/ar.po [hunk 2]
```diff
@@ -3817,7 +3826,8 @@ msgstr "من أجل الحصول على النتائج ذات الصلة، ير
...
+"8695077132796"
```
Comment (teolemon) [inline po/common/ar.po:3830]:
```suggestion
```

File: po/common/ru.po [hunk 3]
```diff
@@ -3185,7 +3193,7 @@ msgstr "Уровень воздействия на окружающую сред
 
 msgctxt "environment_impact_level_example"
 msgid "en:low, en:medium or en:high"
-msgstr "en:низкий, en:средний или en:высокий"
+msgstr "hi"
```
Comment (teolemon) [inline po/common/ru.po:3196]:
```suggestion
msgstr ""
```

File: po/common/ar.po [hunk 4]
```diff
@@ -3573,7 +3581,8 @@ msgstr "استيراد بيانات المنتج"
 # "product photos" in this sentence means data for many products, not just one product
 msgctxt "import_product_photos"
 msgid "Import product photos"
-msgstr "استيراد صور المنتج"
+msgstr "عربي\n"
+"8695077132796"
```
Comment (teolemon) [inline po/common/ar.po:3585]:
```suggestion
msgstr ""
```

**Comment by teolemon** [comment]
#   Failed test 'common: 'on_the_blog_content' in 'zh' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
Wide character in print at /opt/perl/local/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   '<p>要了解有关<site_name>的更多信息，请访问 <a href="https://blog.openfoodfacts.org/en/">我们的博客</a>！</p>
# <p>最近新闻：</p>
# '
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'common: 'text_direction' in 'bn': must be 'ltr' or 'rtl''
#   at tests/unit/i18n.t line 64.
Wide character in print at /opt/perl/local/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   'বাম থেকে ডান'
#     doesn't match '(?^u:^(ltr|rtl)$)'
# Looks like you failed 2 tests of 1777.
tests/unit/i18n.t ............................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/1777 subtests

File: po/common/bn.po [hunk 5]
```diff
@@ -4325,7 +4333,7 @@ msgstr ""
 # Either 'ltr' for left to right languages like English or 'rtl' for right to left languages like Arabic
 msgctxt "text_direction"
 msgid "ltr"
-msgstr ""
+msgstr "বাম থেকে ডান"
```
Comment (teolemon) [inline po/common/bn.po:4336]:
```suggestion
msgstr ""
```

**Comment by teolemon** [comment]
#   Failed test 'product_js_enter_value_between_0_and_max translation in zh contains {max}'
#   at tests/unit/lang.t line 97.
#                   '请输入 {0} 到 {1} 之间的一个值。'
#     doesn't match '(?^u:\{max\})'

File: po/common/en.po [hunk 6]
```diff
@@ -6625,7 +6625,7 @@ msgstr "Number of packaging components"
 
 msgctxt "packagings_materials_all"
 msgid "All materials"
-msgstr "All materials"
+msgstr "Number of packaging components"
```
Comment (teolemon) [inline po/common/en.po:6628]:
```suggestion
msgstr "All materials"
```

Comment (teolemon) [inline po/common/en.po:6628]:
```suggestion
msgstr "All materials"
```

**Pull request merged**